pkg/test/harness.go

```diff
@@ -393,6 +402,16 @@ func (h *Harness) Stop() {
 		}
 	}
 
+	if h.bgProcesses != nil {
+		for _, p := range h.bgProcesses {
+			h.T.Log("killing process ", p)
+			err := p.Process.Kill()
+			if err != nil {
+				h.T.Log("background process kill error", err)
+			}
```

Comment:
Shouldn't we `Wait()` here for the process? 
```Wait waits for the Process to exit, and then returns a ProcessState describing its status and an error, if any. Wait releases any resources associated with the Process. On most operating systems, the Process must be a child of the current process or an error will be returned.```

Reply:
@zen-dog I read the docs...  I'm not sure why we would want to wait... it doesn't add any value at this point

Reply:
The termination of the test harness does not need to wait for other processes outside of its control to exit before exiting.   The harness is killing as we would expect.   Do we want the harness to hang because a user defined process is hanging?     If we were to `wait()` I would rewrite this such that the kill and waits are concurrent and wait for all routines to return.  this seems like a lot of complexity for no gain IMO